flocker/node/test/test_p2p.py

```diff
@@ -171,7 +172,9 @@ def test_uuid(self):
         """
         deployer = self._setup_datasets()
         node_state = self.successResultOf(deployer.discover_state(
-            DeploymentState(nodes={self.EMPTY_NODESTATE}))).node_state
+            DeploymentState(nodes={self.EMPTY_NODESTATE}),
+            persistent_state=PersistentState(),
```

Comment:
:grey_question: Ditto the question about perhaps using a constant in this file for future maintainability.